setup.py

```diff
@@ -34,7 +36,7 @@ def read(*rnames):
 
 setup(
     name='z3c.pagelet',
-    version='2.0.1.dev0',
+    version='3.0.dev0',
```

Comment:
Why 3.0? I think 2.1 is enough as it does not break backwards compatibility. Or am I mistaken?

Reply:
We drop 2.6, but 2.1 seems also fine.